scripts/release.sh

```diff
@@ -133,6 +133,25 @@ git push "${REPOSITORY}" "${RELEASE_TAG}" ${GIT_DRYRUN:-}
 echo "Deploying release to maven central"
 mvn deploy -Prelease -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" -DprocessAllModules=true ${MVN_DEPLOY_DRYRUN:-}
 
+echo "Creating binary distribution"
+mvn package -Pdist -DskipTests=true
+
+if ! command -v gh &> /dev/null
+then
+    echo "gh command could not be found. Please create a release by hand https://github.com/kroxylicious/kroxylicious/releases/new"
+else
+  echo "Creating GitHub release with binary distribution"
+  ORIGINAL_GH_DEFAULT_REPO=$(gh repo set-default -v | (grep -v 'no default repository' || true))
+  gh repo set-default $(git remote get-url ${REPOSITORY})
```

Comment:
maybe we could redundantly do this work setting the default remote higher up and avoid having the duplication between here and lines 172-173